README.md

```diff
@@ -3,7 +3,8 @@
 Light Gradient Boosting Machine
 ===============================
 
-[![GitHub Actions Build Status](https://github.com/microsoft/LightGBM/workflows/R%20GitHub%20Actions/badge.svg?branch=master)](https://github.com/microsoft/LightGBM/actions)
+[![R GitHub Actions Build Status](https://github.com/microsoft/LightGBM/workflows/R%20GitHub%20Actions/badge.svg?branch=master)](https://github.com/microsoft/LightGBM/actions)
```

Comment:
I think we can rename this workflow to simple `R-package`. It will be consistent with other our workflows.
https://github.com/microsoft/LightGBM/blob/415c0cb57794d184ea8639aa5a98a83610639976/.github/workflows/r_package.yml#L1
```suggestion
[![R GitHub Actions Build Status](https://github.com/microsoft/LightGBM/workflows/R-package/badge.svg?branch=master)](https://github.com/microsoft/LightGBM/actions)
```

Reply:
sure

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3726/commits/f886b142b272f099373f5cef7c03661c0c53a8a6 and https://github.com/microsoft/LightGBM/pull/3726/commits/6496d84c1a60d4c9ce0fefcfd19c0ffe0608e130